manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -100,6 +110,19 @@ spec:
             - name: CLUSTER_NAME
               value: kubernetes
           imagePullPolicy: "IfNotPresent"
+          ports:
+            - containerPort: 9808
+              name: healthz
+              protocol: TCP
+          # The probe
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 10
+            timeoutSeconds: 3
+            periodSeconds: 2
```

Comment:
calling probe every 2 seconds is necessary? could you increase the periodic interval?

Reply:
will increase to 5